### ISSUE TITLE
Add ownerReference to all manifests from Helm

### ISSUE THREAD

**Issue by janslow** [Babylonpartners/shipcat#199]
All of the manifests generated by Helm should be linked to the service, by updating each manifest's metadata to include an `ownerReference` pointing to the service's `ShipcatManifest` CRD.

This would simplify deletion of a service and also make manifests easier to visualise.

This is probably blocked on removing tiller.

**Comment by clux** [comment]
Duplicate of https://github.com/Babylonpartners/shipcat/issues/140
